std/machines/large_field/arith256_memory.asm

```diff
@@ -53,7 +54,6 @@ machine Arith256Memory(mem: Memory) with
     // Get an intermediate column that indicates that we're in an
     // actual block, not a default block. Its value is constant
     // within the block.
-    // TODO: Witgen fails if this is an intermediate column.
```

Comment:
Does it not fail any more?

Reply:
oops, it does, I reverted but didn't put back the comment